createMedicCWBenchmark.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Ensure you are on zeebe-cluster
+kubectx gke_zeebe-io_europe-west1-b_zeebe-cluster
+
+# switch do develop
+git checkout develop
+
+# get latest changes
+git fetch
+git pull origin develop
+
+# switch to cw branch
+git checkout medic-cw-benchmarks
+git pull origin medic-cw-benchmarks
+
+# update kw branch
+git merge develop --no-edit
+git push origin medic-cw-benchmarks
+
+# create new kw image and deploy benchmark
+./setupKWBenchmark.sh
+
+# delete older benchmark
+cd benchmarks/setup/
+
+nameOfOldestBenchmark=$(ls | grep medic-cw- | sort | head -n 1)
```

Comment:
:question: Is this risky? Could this cause us to create something with a weird name? :smile:

Reply:
Oehm no :thinking: . Because this is from the existing benchmarks, to delete the existing oldest one.

Reply:
Do you have an alternative?

Reply:
No, I was just wondering what would happen if it grepped the wrong name, if it could disrupt other benchmarks, but I guess the likely hood is low. If it doesn't grep anything then it will fail the command no? It's probably fine :smile: